docs/components/table.md

```diff
@@ -712,7 +860,53 @@ export default {
         'data-testid': 'row-item'
       }
     },
-    defaultSorter
+    defaultSorter,
+    cellAttrsFn ({ headerKey, row, rowIndex, colIndex }) {
+      /**
+       * Sets cell background color based on data returned in
+       * the row parameter and the index of the cell
+       */
+      const backgroundColor = () => {
+        if (row.company && row.company === 'Google') {
+          if (colIndex === 0) {
+            return '#4285F4'
+          } else if (colIndex === 1) {
+            return '#DB4437'
+          } else {
+            return '#F4B400'
+          }
+        }
+
+        return ''
+      }
+
+      /**
+       * Sets cell max-width based on the column headerKey
+       */
+      const colWidth = () => {
+        if (headerKey === 'description') {
+          return '50ch'
+        } else if (headerKey === 'name') {
+          return '22ch'
+        }
+
+        return '25ch'
+      }
+
+      /**
+       * Returns an object of attributes to be applied to cells
+       */
+      return {
+        class: {
+          'truncate': headerKey === 'description' || headerKey === 'name',
+        },
+        'datatest-id': `row-${rowIndex + 1}-col-${headerKey}`,
+        style : {
```

Comment:
```suggestion
        style: {
```
nit: space

Reply:
Fixed in latest commit along with other requested changes, thanks for catching it!